build/optaplanner-build-parent/pom.xml

```diff
@@ -22,6 +22,7 @@
 
     <!-- SNAPSHOT and KIE dependency versions -->
     <version.org.kie.kogito>2.0.0-SNAPSHOT</version.org.kie.kogito>
+    <version.org.drools>7.52.0-SNAPSHOT</version.org.drools> <!-- Must match version used by Kogito. -->
```

Comment:
I don't really like this, but we need it for the alpha network compiler module, which is only in Drools, not in Kogito.

Reply:
The only reason I don't like it is because Drools is part of the KIE 7 release train. The release train architecture is a trainwreck for as long as Drools 8 isn't branched. By making this direct dependency we become a active participant to the situation.

Reply:
Let's standarize the comment on "Keep in sync with"  and add a better breadcrumb to the actual location
```suggestion
    <!-- Keep in sync with version.org.kie7 in kogito-runtimes: https://github.com/kiegroup/kogito-runtimes/blob/master/kogito-build-parent/pom.xml#L185 -->
    <version.org.drools>7.52.0-SNAPSHOT</version.org.drools> <!-- Must match version used by Kogito. -->
```